soh/src/code/z_actor.c

```diff
@@ -2574,7 +2576,11 @@ void Actor_UpdateAll(PlayState* play, ActorContext* actorCtx) {
     if (play->numSetupActors != 0) {
         actorEntry = &play->setupActorList[0];
         for (i = 0; i < play->numSetupActors; i++) {
-            Actor_SpawnEntry(&play->actorCtx, actorEntry++, play);
+            Actor* spawnedActor = Actor_SpawnEntry(&play->actorCtx, actorEntry++, play);
```

Comment:
I wasn't sure if this line should be included in the `#region`, since it was changed and not added. Let me know what the policy is to do here.

Reply:
I don't think it really matters, but I think this is fine as is. If you removed the entire region, the code would function exactly as before the change, so I think in that sense the way it is now is correct. The compiler would warn us about the unused assignment afterwards anyway to clue us in that it also needs to be changed (assuming we see it in the sea of warnings lol.)